pkg/server/plugin/keymanager/awskms/awskms.go

```diff
@@ -377,12 +381,39 @@ func (p *Plugin) fetchAliasesPage(ctx context.Context, marker *string) (*string,
 			if err != nil {
 				return nil, err
 			}
-			p.log.Debug("Loaded key", keyIDTag, *alias.TargetKeyId, aliasTag, *alias.AliasName)
+			p.log.Debug("Key loaded", keyIDTag, *alias.TargetKeyId, aliasTag, *alias.AliasName)
 		}
 	}
 	return aliasesResp.NextMarker, nil
 }
 
+func (p *Plugin) scheduleDeleteTask(ctx context.Context) {
+	t := p.clock.Ticker(30 * time.Second)
```

Comment:
can you add a comment about why ticker is neccesary?

Reply:
I don't think this ticker is necessary. You shouldn't need a ticker to check for the something on the channel.

```
func scheduleDeleteTask(ctx context.Context) {
    for {
        select {
        case <-ctx.Done():
            ... handle context closure ...
        case <-p.scheduleDelete:
            ... handle delete ...
    }
}```

Reply:
I think he added it to prevents a buckle, basically error is send again and again without waiting, maybe we can implement some kind of backoff here?

Reply:
When there is a failure while trying to delete we will push the key once more into the channel. The ticker was a cheap (and probably inelegant) way of doing retries without hitting the API constantly. Even if the SDK includes backoff internally, I fear this might cause a lot of noise in the logs, when there is one key constantly failing. 

Let me know what you think.